webapp/src/app/pacman-features/modules/admin/sticky-exceptions/create-sticky-exceptions/create-sticky-exceptions.component.ts

```diff
@@ -437,7 +434,7 @@ export class CreateStickyExceptionsComponent implements OnInit, OnDestroy {
 
   expiryDate: any;
   getDateData(date: any): any {
-    this.expiryDate = moment(date).format('DD/MM/YYYY');
+    this.expiryDate = this.datePipe.transform(date, 'dd/MM/yyyy');
```

Comment:
We should standardize the date/time format globally. The is awesome, but let's extract the format initially into a constant at the component level, and then we'll define a global constant to make all date/time columns consistent.